### PR TITLE
Some code clean up

### PR DESCRIPTION
**Pull request by jeanlucf22**

### CODE REVIEW

File: src/setup.cc [hunk 1]
```diff
@@ -50,21 +50,27 @@ int MGmol<T>::setupFromInput(const std::string filename)
...
+    Control& ct = *(Control::instance());
+
     // create localization regions
+    Mesh* mymesh           = Mesh::instance();
     const pb::Grid& mygrid = mymesh->grid();
     Vector3D vcell(mygrid.ll(0), mygrid.ll(1), mygrid.ll(2));
     lrs_ = new LocalizationRegions(vcell, ct.tol_orb_centers_move);
 
-    return 0;
+    if (ct.restart_info < 3 || !ct.isLocMode()) setupLRsFromInput(filename);
```
Comment (Rombur) [inline src/setup.cc:67]: This function will give a warning because it doesn't return anything

File: src/ProjectedMatricesSparse.cc [hunk 2]
```diff
@@ -50,12 +49,15 @@ ProjectedMatricesSparse::ProjectedMatricesSparse(
...
+    local_cluster_ = local_cluster;
+
+    Mesh* mymesh             = Mesh::instance();
+    const pb::Grid& mygrid   = mymesh->grid();
+    const pb::PEenv& myPEenv = mymesh->peenv();
+    double domain[3]         = { mygrid.ll(0), mygrid.ll(1), mygrid.ll(2) };
     distributor_invS_
-        = BasicDataDistributors::centeredOrbitalsOverlapDistributor();
-    distributor_sH_ = BasicDataDistributors::orbitalsProdWithHDistributor();
+        = new DataDistribution("invS", lrs_->max_radii(), myPEenv, domain);
```
Comment (Rombur) [inline src/ProjectedMatricesSparse.cc:60]: Do you have a `delete` for this?

Comment (jeanlucf22) [reply src/ProjectedMatricesSparse.cc:60]: fixed now. thanks.

**Comment by jeanlucf22** [comment]
@Rombur I see two failed jobs still hanging for this PR at https://cloud.cees.ornl.gov/jenkins-ci
Could you kill those please? Thanks.

**Comment by Rombur** [comment]
I've killed them. Thanks for letting me know

**Pull request merged**